meta_inspector.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |gem|
   gem.version       = MetaInspector::VERSION
 
   gem.add_dependency 'nokogiri', '~> 1.11.0'
-  gem.add_dependency 'faraday', '~> 1.1.0'
+  gem.add_dependency 'faraday', '>= 1.1.0'
```

Comment:
How about this?

```
gem.add_dependency 'faraday', '~> 1.1'
```

This way we can allow minor releases like 1.4, but avoid major releases like 2.0 that may have incompatible API changes.

Reply:
Great suggestion @jaimeiniesta - also, thanks for #280 🎉